onnx_transformers/pipelines.py

```diff
@@ -555,7 +559,18 @@ def __init__(
             logger.info(f"loading onnx graph from {self.graph_path.as_posix()}")
             self.onnx_model = create_model_for_provider(str(graph_path), "CPUExecutionProvider")
             self.input_names = json.load(open(input_names_path))
-            self.framework = "np"
+            self.framework = "pt"
+            if quantized:
+                onnx_opt_model_path = graph_path.parent.joinpath(f"{graph_path.stem}-opt.onnx")
+                quantized_model_path = graph_path.parent.joinpath(f"{graph_path.stem}-opt-quantized.onnx")
+                if not quantized_model_path.exists() or not onnx_opt_model_path.exists():
+                    opt_options = BertOptimizationOptions('bert')
+                    opt_options.enable_embed_layer_norm = False
+                    onnx_opt_model = optimizer.optimize_model(graph_path.as_posix(),'bert',num_heads=12,hidden_size=768,optimization_options=opt_options)
```

Comment:
we should read `num_heads`, `hidden_size` from `config`, hard coding this will break it for large models